README.md

```diff
@@ -21,27 +21,27 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+The numbers are a timestamp and the purpose of the file is a migration, which helps create tables in your database by modifying the schema.rb file.
 ```
 
 ### Question 3
 
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The router controls which paths your controller has access to
 ```
```

Comment:
:+1: will give you this one but, just to be explicit, the router indicates which controller action a given request will trigger.